server/model.go

```diff
@@ -62,14 +62,26 @@ type chanData struct {
 	data interface{}
 }
 
+// RelatedPlaceRankInfo represents the json structure for rankinglists in the RelatedPlaces cache.
+type RelatedPlaceRankInfo struct {
+	Rank      int32   `json:"rank,omitempty"`
+	Value     float64 `json:"value,omitempty"`
+	PlaceDcid string  `json:"placeDcid,omitempty"`
+}
+
+// RelatedPlaceRanking lists ranking information for the RelatedPlaces cache.
+type RelatedPlaceRanking struct {
+	Info []RelatedPlaceRankInfo `json:"info,omitempty"`
+}
+
 // RelatedPlacesInfo represents the json structure returned by the RelatedPlaces cache.
 type RelatedPlacesInfo struct {
```

Comment:
can remove these go structs?

Reply:
i left them for completeness, but you're right, they're not used. removed.